src/index.html

```diff
@@ -6,9 +6,70 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img src="images/logo.png"
+        alt="logo"
+        class="logo__img"
+        >
```

Comment:
![Screenshot 2023-01-25 at 15 45 55](https://user-images.githubusercontent.com/84275496/214636800-4b11339a-b513-4df9-8cca-8d7bb820cb05.png)

Reply:
Pay attention to code style